cardano-constitution/src/Cardano/Constitution/Data/Validator.hs

```diff
@@ -1,26 +1,28 @@
 -- editorconfig-checker-disable-file
 {-# LANGUAGE OverloadedLists #-}
-module Cardano.Constitution.Data.Validator
-    ( module Export
-    , defaultValidators
-    , defaultValidatorsWithCodes
-    ) where
+
+module Cardano.Constitution.Data.Validator (
+  module Export,
+  defaultValidators,
+  defaultValidatorsWithCodes,
+) where
```

Comment:
Change from leading commas to trailing? 

Ok, I personally prefer leading, but don't mind trailing in the case of imports.